index.html

```diff
@@ -58,13 +60,17 @@ <h1 id="title"> {{ appCtrl.Event.name }} </h1>
     <script src="app.js"></script>
     <script src="config.js"></script>
     <script src="app/components/common/ApiJsonFactory.js"></script>
+    <script src="app/components/common/TwitterJsonFactory.js"></script>
     <script src="app/components/map/MapController.js"></script>
     <script src="app/components/sessions/SessionsController.js"></script>
     <script src="app/components/sidenav/SidenavController.js"></script>
     <script src="app/components/speakers/SpeakersController.js"></script>
     <script src="app/components/sponsors/SponsorsController.js"></script>
-    <script src="app/components/tracks/TracksController.js"></script>
     <script src="app/components/twitterwall/TwitterwallController.js"></script>
+    <script src="app/components/twitterwall/background-directive.js"></script>
+    <script src="app/components/twitterwall/singletweet.js"></script>
```

Comment:
@championswimmer ,

Help me in this ,seems they are already there
![1](https://cloud.githubusercontent.com/assets/12194719/13553941/0877fbe4-e3c0-11e5-8dda-e2dae8f388c3.png)